trackintel/geogr/distances.py

```diff
@@ -161,4 +162,17 @@ def meters_to_decimal_degrees(meters, latitude):
     float
         An approximation of a distance (given in meters) in degrees.
     """
-    return meters / (111.32 * 1000.0 * cos(latitude * (pi / 180.0)))
\ No newline at end of file
+    return meters / (111.32 * 1000.0 * cos(latitude * (pi / 180.0)))
+
+
+def check_crs(data):
+    wgs = False
+    if data.crs == 4326:
+        wgs=True
+        warnings.warn('Your data is in WGS84, for length calculation the haversine distance is used')
```

Comment:
Hm... if the data is in wgs84 than using haversine seems like the expected behavior. In this case we should not throw a warning.